src/prefect/cli/worker.py

```diff
@@ -156,5 +157,13 @@ async def start(
 
             started_event = await worker._emit_worker_started_event()
 
+            config = uvicorn.Config(
+                "prefect.workers.server.app",
+                port=8080,
+                log_level="error",
+            )
+            server = uvicorn.Server(config)
+            await server.serve()
```

Comment:
im not sure if we want to be blocking here, but i lifted this pattern from the uvicorn docs

Reply:
also not sure about uvicorn's behavior on the main thread, in general

Reply:
This will take over all signal handlers afaik which would probably be an issue

Reply:
https://github.com/encode/uvicorn/issues/1579